units/en/unit1/rl-framework.mdx

```diff
@@ -0,0 +1,146 @@
+
+## Observations/States Space [[obs-space]]
+
+Observations/States are the **information our agent gets from the environment.** In the case of a video game, it can be a frame (a screenshot). In the case of the trading agent, it can be the value of a certain stock, etc.
+
+There is a differentiation to make between *observation* and *state*:
+
+- *State s*: is **a complete description of the state of the world** (there is no hidden information). In a fully observed environment.
+
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/chess.jpg" alt="Chess">
+<figcaption>In chess game, we receive a state from the environment since we have access to the whole check board information.</figcaption>
+</figure>
+
+In chess game, we receive a state from the environment since we have access to the whole check board information.
+
+With a chess game, we are in a fully observed environment, since we have access to the whole check board information.
+
+- *Observation o*: is a **partial description of the state.** In a partially observed environment.
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/mario.jpg" alt="Mario">
+<figcaption>In Super Mario Bros, we only see a part of the level close to the player, so we receive an observation.</figcaption>
+</figure>
+
+In Super Mario Bros, we only see a part of the level close to the player, so we receive an observation.
+
+In Super Mario Bros, we are in a partially observed environment. We receive an observation **since we only see a part of the level.**
+
+<Tip>
+In reality, we use the term state in this course but we will make the distinction in implementations.
+</Tip>
+
+To recap:
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/obs_space_recap.jpg" alt="Obs space recap" width="100%">
+
+
+## Action Space [[action-space]]
+
+The Action space is the set of **all possible actions in an environment.**
+
+The actions can come from a *discrete* or *continuous space*:
+
+- *Discrete space*: the number of possible actions is **finite**.
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/mario.jpg" alt="Mario">
+<figcaption>Again, in Super Mario Bros, we have only 4 directions and jump possible</figcaption>
+</figure>
+
+In Super Mario Bros, we have a finite set of actions since we have only 4 directions and jump.
+
+- *Continuous space*: the number of possible actions is **infinite**.
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/self_driving_car.jpg" alt="Self Driving Car">

```

Comment:
Do we need to add a reference for the source of the image?

Reply:
It was an open source Pixabay image so normally no